### PR TITLE
RPET-50: Remove solicitor cheque related payment fields

### PR DESCRIPTION
**Pull request by iichr**
## Change description
- Remove:
  - the option to pay by cheque from the petitioner solicitor payment dropdown
  - the `What to do next` section on the `SolPaymentSummary` page which had to do with cheques, and the address to post them to
  - related field permissions

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPET-50

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseEvent.json [hunk 1]
```diff
@@ -2516,8 +2516,8 @@
     "DisplayOrder": 1,
     "PreConditionState(s)": "AwaitingService",
     "PostConditionState": "AosAwaiting",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/bulk-print",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/bulk-print",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent.json]: ? what is changed?

File: definitions/divorce/json/FixedLists.json [hunk 2]
```diff
@@ -581,12 +581,6 @@
     "ListElementCode": "feesHelpWith",
     "ListElement": "Help with fees"
   },
-  {
-    "LiveFrom": "01/01/2017",
```
Comment (yarpo) [inline definitions/divorce/json/FixedLists.json]: what about existing cases that already have "cheque"?

Comment (iichr) [reply definitions/divorce/json/FixedLists.json]: I had that question too before I started 🤷‍♂  need to clarify if there are indeed no such cases

File: definitions/divorce/json/CaseEvent.json (hunk 1 above)
Comment (iichr) [reply definitions/divorce/json/CaseEvent.json]: Generating the json from excel changed the ordering of the fields, it's automatic

File: definitions/divorce/json/FixedLists.json (hunk 2 above)
Comment (yarpo) [reply definitions/divorce/json/FixedLists.json]: do you know if we would be able to upload this on prod if there are any cheques?

Comment (iichr) [reply definitions/divorce/json/FixedLists.json]: Just checked on prod, there are only 11 cases in total that use the cheque option currently, last one modified is today, the 6th Feb. Updating ticket with details and pinging Izzy

Comment (yarpo) [reply definitions/divorce/json/FixedLists.json]: ok, so we need to do data migration. Apparently @HarryH96 knows how.

**Pull request merged**